amida-tech/indaba-client
# Bug/INBA-976 Force task complete not updating state

#### What does this PR do?
Adjusts the force complete function of the task options modal to properly update the state when a task is set to "force complete" and thus render the completed stage.

#### Related JIRA tickets:

INBA 976

#### How should this be manually tested?

**MAKE SURE YOU HAVE THE NEWEST VERSION OF GREYSCALE, OR THIS WILL NOT WORK**

Log into a project that has several stages. Open task options on an assigned stage and set it to force complete. The completed task should now show the "done" tag after the modal closes. 

Also test by creating a new stage, assigning a user, and then setting to force complete.
